pennylane/devices/qubit/initialize_state.py

```diff
@@ -42,4 +42,4 @@ def create_initial_state(
         state[(0,) * num_wires] = 1
         return qml.math.asarray(state, like=like)
 
-    return qml.math.asarray(prep_operation.state_vector(wire_order=list(wires)), like=like)
+    return qml.math.asarray(prep_operation.state_vector(), like=like)
```

Comment:
@mudit2812 , I could not delete `wire_order` in my `state_vector` because it is called here. Is it ok if I delete `wire_order` here?